python/py-blis/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                                      1.0
+PortGroup           python                      1.0
+PortGroup           github                      1.0
+
+name                py-blis
+version             0.3.0
+revision            0
+github.setup        explosion cython-blis ${version} v
+
+checksums           rmd160  2eb5c8a0e13a8b8fb2095ba9ace21236c130a956 \
+                    sha256  6331325269afd959c57c093bf1c9f02eb6be28cbcacc9279d6de4436464cc90f \
+                    size    1663871
+
+platforms           darwin
+supported_archs     x86_64
+
+license             MIT
+
+maintainers         {jonesc @cjones051073} openmaintainer
+
+description         Fast matrix-multiplication as a self-contained Python library
+long_description    ${description}
+
+github.livecheck.regex  {([0-9.]+)}
+
+# Support python versions
+python.versions     35 36 37
+
+if {${name} ne ${subport}} {
+
+    depends_build-append \
+        port:py${python.version}-setuptools \
+        port:py${python.version}-wheel
+
+    depends_lib-append \
+        port:py${python.version}-cython \
+        port:py${python.version}-pytest \
```

Comment:
```pytest``` and ```hypothesis``` are test dependencies and not needed for using the library, so they shouldn't be added to ```depends_lib```. They are listed in ```requirements.txt``` (presumably for testing or development use), but are  not in ```setup.py``` as ```install_requires```.

Reply:
ah yes, my mistake...